src/System.Linq.Expressions/tests/BinaryOperators/Arithmetic/BinaryAddTests.cs

```diff
@@ -282,6 +336,54 @@ private static void VerifyShortAdd(short a, short b)
             }
         }
 
+        private static void VerifyShortAddOvf(short a, short b)
```

Comment:
There's a lot of repetition across these tests, which all follow the same pattern and seem like they could be done simply by making a and b generic, and by having the method accept an add function as a delegate.